kofta/src/vscode-webview/components/BottomVoiceControl.tsx

```diff
@@ -97,6 +98,7 @@ export const BottomVoiceControl: React.FC<BottomVoiceControlProps> = ({
         key="invite"
         onClick={() => {
           wsend({ op: "fetch_invite_list", d: { cursor: 0 } });
+          if (open) toggleOpen();
```

Comment:
why here?

Reply:
To close chat when entering the invite screen.